spec/keycard/request/attributes_spec.rb

```diff
@@ -4,104 +4,113 @@
 require "keycard/request/attributes"
 
 RSpec.describe Keycard::Request::Attributes do
-  let(:inst_attributes) { {} }
+  let(:finder_attributes) { { baz: 'quux', finder_attr: 'value' } }
   let(:finder) do
-    double(:finder, attributes_for: inst_attributes, identity_keys: [:inst_attr])
+    double(:finder, attributes_for: finder_attributes, identity_keys: [:finder_attr])
   end
-
-  let(:user_attributes)    { { username: 'user' } }
-  let(:request)            { double(:request, attributes: user_attributes, client_ip: '10.0.0.1') }
-  let(:request_attributes) { described_class.new(request, finders: [finder]) }
+  let(:auth_token) { "myauthtoken" }
+  let(:rack_request) do
+    double(
+      :rack_request,
+      env: { 'HTTP_AUTHORIZATION' => "Token token=\"#{auth_token}\", opt=\"someopt\""}
+    )
+  end
+  let(:attributes) { described_class.new(rack_request, finders: [finder]) }
 
   it "takes a request" do
-    expect(request_attributes).not_to be(nil)
+    expect(attributes).not_to be(nil)
   end
 
-  it "lists combines its identity keys with those of its finders" do
-    expect(request_attributes.identity_keys).to contain_exactly(:user_pid, :user_eid, :inst_attr)
+  describe "#identity_keys" do
+    it "includes user_pid and user_eid by default" do
+      expect(described_class.new(double(:r), finders: []).identity_keys)
+        .to contain_exactly(:user_pid, :user_eid)
+    end
+    it "lists combines its identity keys with those of its finders" do
+      expect(attributes.identity_keys).to contain_exactly(:user_pid, :user_eid, :finder_attr)
+    end
   end
 
-  it "does not give a value for user_pid" do
-    expect(request_attributes.user_pid).to be nil
+  describe "#user_pid" do
+    it { expect(attributes.user_pid).to be nil }
```

Comment:
That seems confusing for anyone looking at the specs who doesn't already know that `specify` is just an alias for `it`. I had to look it up because I had never seen it before.

Reply:
Yeah, I guess my brain just threw a grammar exception. Consider it caught/logged/discarded in favor of familiarity.